src/functions-and-arrays.js

```diff
@@ -78,8 +155,17 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(arr, word) {
```

Comment:
Take a further look on this function, it is not passing the tests yet.
The logic is good, but there are 2 things you need to change to make it work

- if you do `sum += sum[i]`your are adding the string element on the sum variable, what you want instead is to `sum += 1` every time a word is found.
- The function return is inside the for loop, it should be outside at the end of your function